living_documentation_generator/github_projects.py

```diff
@@ -61,69 +61,55 @@ def send_graphql_query(self, query: str) -> dict[str, dict]:
         except Exception as e:
             logger.error("An error occurred: %s.", e, exc_info=True)
 
-        return {}
+        return
 
     def get_repository_projects(self, repository: Repository, projects_title_filter: list[str]) -> list[GithubProject]:
         projects = []
 
         # Fetch the project response from the GraphQL API
-        projects_from_repo_query = GithubProjectQueries.get_projects_from_repo_query(repository.owner.login,
-                                                                                     repository.name)
+        projects_from_repo_query = GithubProjectQueries.get_projects_from_repo_query(organization_name=repository.owner.login,
+                                                                                     repository_name=repository.name)
 
-        projects_from_repo_response = self.send_graphql_query(projects_from_repo_query)
+        projects_from_repo_response = self.__send_graphql_query(projects_from_repo_query)
+
+        if projects_from_repo_response is None:
+            logger.warning("Project response is None for repository `%s`.", repository.full_name)
+            return projects
 
         # If response is not None, parse the project response
         if projects_from_repo_response['repository'] is not None:
             projects_from_repo_nodes = projects_from_repo_response['repository']['projectsV2']['nodes']
-            repository_projects = []
 
             for project_json in projects_from_repo_nodes:
                 # Check if the project is required based on the configuration filter
                 project_title = project_json['title']
+                project_number = project_json['number']
 
                 # If no filter is provided, all projects are required
-                is_project_required = True if len(projects_title_filter) else project_title in projects_title_filter
+                is_project_required = True if not projects_title_filter else project_title in projects_title_filter
 
                 # Main project structure is loaded and added to the projects list
                 if is_project_required:
```

Comment:
Add debug message to inform that project has been removed from fetching by config defined filter.

Reply:
Added in commit: [bd3a2de](https://github.com/AbsaOSS/living-doc-generator/pull/27/commits/bd3a2dedfc922c6844173c84b1d0040cb41286b7)